packages/components/inputs/select-input/src/select-input.tsx

```diff
@@ -44,29 +44,33 @@ export type TCustomEvent = {
   persist: () => void;
 };
 
+export type THorizontalConstraint =
```

Comment:
Do we actually want to support all horizontal constraints for both min and max props?
I doesn't seem the 1 value, for instance to be allowed for the max prop.

Reply:
@ddouglasz has this already been addressed?